api/api_sources/sources/server/modules/observation/observation.route.ts

```diff
@@ -27,6 +28,36 @@ export class ObservationRouteController extends ResourceRouteController<Observat
     static get shared(): ObservationRouteController {
         return this.sharedInstance() as ObservationRouteController;
     }
+
+    @Get({
+        path: '/search',
+    })
+    public async search(req: any, data?: any): Promise<[number, any]> {
```

Comment:
Move this method to ObservationController
Route Controller should handle only route related functions and integrations of the different data controllers.